outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
+  3. No longer use `api.exports()` in packages (?), publish to NPM if not Meteor specific (?)
```

Comment:
We have an answer for this, will write the version checking package and document it here.